### DISCUSSION TITLE
Adjustable font size and digits of precision for raw data and indicator panels

### DISCUSSION THREAD

**Discussion by vivek-rane** [foxglove/community#681] [Q&A]
For all indicator panels and raw data panels, it would be great if we could adjust the font size in relation to the current zoom level.

It would also be great if we could round to a specified number of digits in raw data panels when displaying a single value.

Without these features, this is what it looks like when I switch from a large monitor to my laptop screen
<img width="194" alt="image" src="https://user-images.githubusercontent.com/25373098/230183873-ee7b64c3-9f7c-4dc1-baff-9464e5744963.png">

**Comment by jtbandes** [comment]
See also:
- https://github.com/foxglove/community/discussions/620

**Comment by sharst** [comment]
I would like to thrown in my hat for this one - without this feature it is almost impossible to have several indicator panels open. 
I would vote for making the text statically configurable, this makes it easier to have several panels with the same text size, which IMO would give a more orderly appearance.

**Comment by sharst** [comment]
I would like to thrown in my hat for this one - without this feature it is almost impossible to have several indicator panels open. 
I would vote for making the text statically configurable, this makes it easier to have several panels with the same text size, which IMO would give a more orderly appearance.

**Comment by defunctzombie** [comment]
The raw message panel already has a font size setting:

<img width="242" alt="image" src="https://github.com/foxglove/community/assets/84792/391b9ad0-183b-4e02-b3cc-4dfdc8a274ed">
